### YamlMime:PythonClass
uid: azure.iot.device.exceptions.ConnectionDroppedError
name: ConnectionDroppedError
fullName: azure.iot.device.exceptions.ConnectionDroppedError
module: azure.iot.device.exceptions
inheritances:
- azure.iot.device.exceptions.ClientError
summary: Lost connection while executing operation
constructor:
  syntax: ConnectionDroppedError()
methods:
- uid: azure.iot.device.exceptions.ConnectionDroppedError.add_note
  name: add_note
  summary: "Exception.add_note(note) \u2013\nadd a note to the exception"
  signature: add_note()
- uid: azure.iot.device.exceptions.ConnectionDroppedError.with_traceback
  name: with_traceback
  summary: "Exception.with_traceback(tb) \u2013\nset self.__traceback__ to tb and\
    \ return self."
  signature: with_traceback()
attributes:
- uid: azure.iot.device.exceptions.ConnectionDroppedError.args
  name: args
